client/src/components/OnboardingCredentials/OnboardingCredentials.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "./OnboardingCredentials.css";
+import InputTextField from "../InputTextField/InputTextField.js";
+import Button from "../Button/Button.js";
+
+function OnboardingCredentials(props) {
+  return (
+    <div className={props.classList}>
+      <form className="OnboardingCredentials__form" action="">
+        <h1 className="OnboardingCredentials__header">Step One</h1>
+        <div className="OnboardingCredentials__container">
+          <InputTextField type="text" name="name" placeholder="Name"></InputTextField>
```

Comment:
How will we pass the values of these inputs back up? Do we need callbacks or something?